src/pmpo_MPMesh.cpp

```diff
@@ -282,7 +357,7 @@ void MPMesh::printVTP_mesh(int printVTPIndex){
     FILE * pFile = fopen(fileOutput,"w");
     free(fileOutput);
 
-    DoubleVec3dView::HostMirror h_vtxCoords = Kokkos::create_mirror_view(vtxCoords);
+    auto h_vtxCoords = Kokkos::create_mirror_view(vtxCoords);
     IntVtx2ElmView::HostMirror h_elm2VtxConn = Kokkos::create_mirror_view(elm2VtxConn);
```

Comment:
The data type is IntVtx2ElmView but copies elm2VtxConnection (maybe a naming issue?)